client/src/components/homePageTabs/HomePageTabs.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import {NavLink} from 'react-router-dom';
+import styled from 'styled-components';
+
+export default function HomePageTabs() {
+  return (
+    <Wrapper>
+        
+        <NavLink to="/"><Button>Anstehend</Button></NavLink>
```

Comment:
I don't think that you need the button here; you can style the NavLink as well!

```suggestion
        <NavLink to="/">Anstehend</NavLink>
```

Reply:
You're totally right!